ops/_private/harness.py

```diff
@@ -316,8 +316,8 @@ def __init__(
         )
 
         warnings.warn(
-            'Harness is deprecated; we recommend using state transition testing '
-            "(previously known as 'Scenario') instead",
+            'Harness is deprecated. See more: '
```

Comment:
I'm being picky, but the how-to page won't actually give any more information about Harness being deprecated :wink: 

How about:

```suggestion
            'Harness is deprecated. For the recommended approach, see: '
```